gravitee-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/GroupServiceImpl.java

```diff
@@ -517,13 +531,19 @@ public void delete(String groupId) {
                         application.setUpdatedAt(updatedDate);
                         try {
                             applicationRepository.update(application);
+                            applicationIds.add(application.getId());
                         } catch (TechnicalException ex) {
                             logger.error("An error occurs while trying to delete a group", ex);
                             throw new TechnicalManagementException("An error occurs while trying to delete a group", ex);
                         }
                     }
                 );
 
+            eventManager.publishEvent(
+                ApplicationAlertEventType.APPLICATION_MEMBERSHIP_UPDATE,
+                new ApplicationAlertMembershipEvent(applicationIds, Collections.emptySet())
```

Comment:
why not not using groupId here ?

Reply:
in this case I have to use the applicationIds because:
- by its async nature, the group will probably already be deleted when I will try to query applicationService.findByGroup()
- since we already get all the applications in this method, it avoids to call the db again